js/admin.block-post.js

```diff
@@ -34,6 +34,32 @@
 			$( '#block-properties-icon-current' ).html( svg );
 		});
 
+		$( '#block_template .template-location a.filename' ).on( 'click', function( event ) {
+			event.preventDefault();
+
+			let copy  = $( '#block_template .template-location .click-to-copy' ),
```

Comment:
No need to change this, but I've lately been using `const` instead of `let` whenever the variable won't be reassigned, like here.

Reply:
I'll leave it as is in this instance, for consistency with the rest of the script.